emf/src/main/java/com/powsybl/tutorials/emf/BalancesAdjustmentValidationParameters.java

```diff
@@ -31,7 +32,10 @@ public static BalancesAdjustmentValidationParameters load() {
 
         // Read parameters under balances-adjustment-validation-parameters module
         PlatformConfig platformConfig = PlatformConfig.defaultConfig();
-        ModuleConfig config = platformConfig.getModuleConfig("balances-adjustment-validation-parameters");
+        ModuleConfig config = platformConfig.getOptionalModuleConfig("balances-adjustment-validation-parameters").orElse(null);
+        if (config == null) {
+            throw new PowsyblException("Could not find parameters for balances adjustment validation.");
+        }
```

Comment:
You can also use the `Optional.orElseThrow(...)` method, to avoid an unnecessary condition:
```suggestion
        ModuleConfig config = platformConfig.getOptionalModuleConfig("balances-adjustment-validation-parameters")
                .orElseThrow(() -> new PowsyblException("Could not find parameters for balances adjustment validation."));
```

Reply:
Thanks !